requirements/requirements_git.txt

```diff
@@ -3,3 +3,4 @@ git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
 git+https://github.com/ansible/python3-saml.git@devel#egg=python3-saml
 django-ansible-base @ git+https://github.com/ansible/django-ansible-base@devel#egg=django-ansible-base[rest_filters,jwt_consumer,resource_registry,rbac]
+awx-plugins-core @ git+https://git@github.com/chrismeyersfsu/awx-plugins.git@entrypoints2#awx-plugins-core
```

Comment:
This should be changed to roughly `awx-plugins-core @ git+https://git@github.com/ansible/awx-plugins.git@devel#awx-plugins-core` before merging.

Reply:
`#awx-plugins-core` is incorrect. The syntax should be `#egg=awx-plugins-core`